dev/com.ibm.ws.cdi.extension_fat/test-applications/SPIExtension.war/src/com/ibm/ws/cdi/extension/spi/test/app/CustomBDABeanTwo.java

```diff
@@ -16,7 +16,7 @@
 public class CustomBDABeanTwo {
 
     public String toString() {
-        return "CustomBDABeanTwo";
+        return "A Bean with an annotation registered via getBeanDefiningAnnotationClasses was successfully injected into a different bean with an annotation registered via getBeanDefiningAnnotationClasses";
```

Comment:
I think it is good practice to make strings like these into constants ... so that the results can be checked directly rather that risking a typo in the comparison.